code/obj/flock/structure/relay.dm

```diff
@@ -142,37 +141,38 @@
 	logTheThing(LOG_GAMEMODE, src, "Flock relay[src.flock ? " belonging to flock [src.flock.name]" : ""] unleashes the signal, exploding at [log_loc(src)].")
 	src.finished = TRUE
 	src.flock.relay_finished = TRUE
-	processing_items -= src
 	var/turf/location = get_turf(src)
 	overlays += "structure-relay-sparks"
 	desc = "Your life is flashing before your eyes. Looks like this is the end."
 	flock_speak(null, "!!! TRANSMITTING SIGNAL !!!", src.flock)
 	src.visible_message("<span class='flocksay bold'>[src] begins sparking wildly! The air is charged with static!</span>")
-	for(var/mob/M in mobs)
-		M.playsound_local(M, 'sound/misc/flockmind/flock_broadcast_charge.ogg', 30, 0)
-	sleep(final_charge_time_length SECONDS)
 
-	for(var/mob/M in mobs)
-		M.playsound_local(M, 'sound/misc/flockmind/flock_broadcast_kaboom.ogg', 30, 0)
-		M.flash(3 SECONDS)
-	if (!src.shuttle_departure_delayed)
-		SPAWN(1 SECOND)
-			emergency_shuttle.disabled = SHUTTLE_CALL_ENABLED
-			emergency_shuttle.incall()
-			emergency_shuttle.can_recall = FALSE
-			emergency_shuttle.settimeleft(60) // cut the time down to keep some sense of urgency
-			boutput(world, "<span class='notice'><B>Alert: The emergency shuttle has been called.</B></span>")
-			boutput(world, "<span class='notice'>- - - <b>Reason:</b> Hostile transmission intercepted. Sending rapid response emergency shuttle.</span>")
-			boutput(world, "<span class='notice'><B>It will arrive in [round(emergency_shuttle.timeleft())] seconds.</B></span>")
-	sleep(2 SECONDS)
-	for(var/x = -2 to 2)
-		for(var/y = -2 to 2)
-			flockdronegibs(locate(location.x + x, location.y + y, location.z))
-	explosion_new(src, location, 2000)
-	gib(location)
-	flock_signal_unleashed = TRUE
-	sleep(2 SECONDS) //allow them to hear the explosion before their headsets scream and die
-	destroy_radios()
+	SPAWN(0)
```

Comment:
maybe just call the proc in a `SPAWN` instead?

Reply:
If you mean the overall proc, it was just setup this way so that important stuff could immediately happen, not sure if there would be anything bad from creating repeated spawns through process()